torch_geometric/loader/link_neighbor_loader.py

```diff
@@ -170,6 +170,7 @@ class LinkNeighborLoader(LinkLoader):
             negative sampling mode.
             If set to :obj:`None`, no negative sampling strategy is applied.
             (default: :obj:`None`)
+            For example use obj:`neg_sampling=dict(mode= 'binary', amount=0.5)`
```

Comment:
@puririshi98 The formatting is broken. Mind sending a quick PR?
```suggestion
            For example, use :obj:`neg_sampling=dict(mode='binary', amount=0.5)`
```